apps/meteor/app/livechat/server/lib/RoutingManager.ts

```diff
@@ -199,6 +199,10 @@ export const RoutingManager: Routing = {
 
 		const { servedBy } = room;
 
+		if (shouldQueue) {
```

Comment:
pls create a test case for this: `Manager/Monitor forwarding a chat from the queue to an offline department`